src/sentry/interfaces/stacktrace.py

```diff
@@ -411,8 +411,8 @@ def get_hash(self):
                 output.append('<function>')
             else:
                 output.append(remove_function_outliers(self.function))
-        elif self.lineno is not None:
-            output.append(self.lineno)
+            if self.lineno is not None:
```

Comment:
This will break heuristics in other languages where both a context line *and* a lineno is present, which we dont want.

Basically any change to the heuristics has to be thoroughly evaluated (and at this stage, requires tests). I dont have enough context of the case involved here, but this change as is cant be accepted.